final_task/rss_reader/rss_parser.py

```diff
@@ -0,0 +1,325 @@
+import feedparser
+import json
+import logging
+import psycopg2
+from yattag import Doc
+from database_functions import checkExistance
+from database_functions import createTable
+from database_functions import putIntoDB
+from database_functions import readingPasswordFromFile
+
+class RssParser():
+    """ Parses news from website"""
+
+    url = ""
+    limit = 0
+    date = None
+    jsonFormat = False
+    path = None
+    to_html = False
+
+    def __init__(self, parametrUrl, parametrLimit, parametrJsonFormat = False, parametrDate = None, parametrPath = None,
+                 parametrHtml = False):
+        """ Inits arguments"""
+
+        self.url = parametrUrl
+        self.limit = parametrLimit
+        self.jsonFormat = parametrJsonFormat
+        self.date = parametrDate
+        self.to_html = parametrHtml
+        self.path = parametrPath
+        logging.debug("Check in parser")
+
+        if self.date != None:
+            self.printCashe()
+        else:
+            self.parse()
+
+    def jsonFromCashe(self, rows):
+        """ Convets news from cache in json format"""
+
+        listToJsonFormat = []
+        for row in rows:
+            if len(row[4]) > 1:
+                dictionary = {"Title": row[1],
+                              "Date": row[2],
+                              "Description": row[3],
+                              "Link [1]": row[4][0],
+                              "Link [2]": row[4][1]}
+            else:
+                dictionary = {"Title": row[1],
+                              "Date": row[2],
+                              "Description": row[3],
+                              "Link [1]": row[4][0]}
+            listToJsonFormat.append(dictionary)
+        jsonData = json.dumps(listToJsonFormat, indent=5, ensure_ascii=False)
+        print(jsonData)
+        return None
```

Comment:
You do not really need to write `return None` explicitly at the end of the function